internal/pkg/skuba/kubeadm/configmap.go

```diff
@@ -62,6 +62,14 @@ func GetCurrentClusterVersion() (*version.Version, error) {
 	return version.MustParseSemantic(initCfg.KubernetesVersion), nil
 }
 
+// GetKubeadmApisVersion returns the api version to use in the kubeadm-init.conf
+func GetKubeadmApisVersion(kubernetesVersion *version.Version) (string, error) {
```

Comment:
This function never returns any error, so maybe we should just return one value?

Reply:
oops indeed, forgot to update this. thanks for the hint